File: JS_front_end/Functions_statements/Exercise/04. Odd And Even Sum.js
function oddeven(number) {
    let numString = number.toString();
    let oddsum = 0;
    let evensum = 0;
    for (let i = 0; i < numString.length; i++) {
        let digit = parseInt(numString[i]);
        if (digit % 2 === 0) {
            evensum += digit;
        }
        else {
            oddsum += digit;
        }
    }
    console.log(`Odd sum = ${oddsum}, Even sum = ${evensum}`);
}

File: Fundamentals/Functions - Exercise/04. Odd and Even Sum.py
def odd_or_even(a):
    odd_sum = 0
    even_sum = 0
    x = [int(c) for c in str(a)]
    for current_index in x:
        if current_index % 2 == 0:
            even_sum += current_index
        else:
            odd_sum += current_index
    return f'Odd sum = {odd_sum}, Even sum = {even_sum}'

number = int(input())
print(odd_or_even(number))
